# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1422569596 18000
#      Thu Jan 29 17:13:16 2015 -0500
# Node ID 3b0b4d55002dba8dfdcc3db41712ed4655765cf2
# Parent  61cc00ebac60024d32a6bc7b38f27e7ddad98b9f
rename __fltk_print__.m to __opengl_print__.m

* __opengl_print__.m: Rename from __fltk_print__.m.
* scripts/plot/util/module.mk (plot_util_PRIVATE_FCN_FILES):
Update list.
* print.m: Call __opengl_print__.m instead of __fltk_print__.m.

diff --git a/scripts/plot/util/module.mk b/scripts/plot/util/module.mk
--- a/scripts/plot/util/module.mk
+++ b/scripts/plot/util/module.mk
@@ -3,27 +3,27 @@ FCN_FILE_DIRS += plot
 plot_util_GEN_FCN_FILES = \
   plot/util/gnuplot_binary.m
 
 GEN_FCN_FILES += $(plot_util_GEN_FCN_FILES)
 
 plot_util_PRIVATE_FCN_FILES = \
   plot/util/private/__add_default_menu__.m \
   plot/util/private/__fltk_ginput__.m \
-  plot/util/private/__fltk_print__.m \
   plot/util/private/__ghostscript__.m \
   plot/util/private/__gnuplot_get_var__.m \
   plot/util/private/__gnuplot_ginput__.m \
   plot/util/private/__gnuplot_has_feature__.m \
   plot/util/private/__gnuplot_has_terminal__.m \
   plot/util/private/__gnuplot_open_stream__.m \
   plot/util/private/__gnuplot_print__.m \
   plot/util/private/__gnuplot_version__.m \
   plot/util/private/__go_draw_axes__.m \
   plot/util/private/__go_draw_figure__.m \
+  plot/util/private/__opengl_print__.m \
   plot/util/private/__print_parse_opts__.m \
   plot/util/private/__tight_eps_bbox__.m
 
 plot_util_FCN_FILES = \
   plot/util/__actual_axis_position__.m \
   plot/util/allchild.m \
   plot/util/ancestor.m \
   plot/util/axes.m \
diff --git a/scripts/plot/util/print.m b/scripts/plot/util/print.m
--- a/scripts/plot/util/print.m
+++ b/scripts/plot/util/print.m
@@ -423,17 +423,17 @@ function print (varargin)
       endif
     endif
 
     ## call the graphics toolkit print script
     switch (get (opts.figure, "__graphics_toolkit__"))
       case "gnuplot"
         opts = __gnuplot_print__ (opts);
       otherwise
-        opts = __fltk_print__ (opts);
+        opts = __opengl_print__ (opts);
     endswitch
 
   unwind_protect_cleanup
     ## restore modified properties
     if (isstruct (props))
       ## Restore figure position and units first
       for n = 2:-1:1
         if (ishandle (props(n).h))
diff --git a/scripts/plot/util/private/__fltk_print__.m b/scripts/plot/util/private/__opengl_print__.m
rename from scripts/plot/util/private/__fltk_print__.m
rename to scripts/plot/util/private/__opengl_print__.m
--- a/scripts/plot/util/private/__fltk_print__.m
+++ b/scripts/plot/util/private/__opengl_print__.m
@@ -12,21 +12,21 @@
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
-## @deftypefn {Function File} {} __fltk_print__ (@var{@dots{}})
+## @deftypefn {Function File} {} __opengl_print__ (@var{@dots{}})
 ## Undocumented internal function.
 ## @end deftypefn
 
-function opts = __fltk_print__ (opts)
+function opts = __opengl_print__ (opts)
 
   dos_shell = (ispc () && ! isunix ());
 
   set (0, "currentfigure", opts.figure);
   drawnow ("expose");
 
   if (! isempty (opts.fig2dev_binary))
     ## fig2dev is prefered for conversion to emf
@@ -155,17 +155,17 @@ function opts = __fltk_print__ (opts)
   endif
   is2D = all (abs (vw(:,2)) == 90);
   if (is2D)
     gl2ps_device{end} = [gl2ps_device{end}, "is2D"];
   endif
 
   for n = 1:numel (pipeline)
     if (opts.debug)
-      fprintf ("fltk-pipeline: '%s'\n", pipeline{n});
+      fprintf ("opengl-pipeline: '%s'\n", pipeline{n});
     endif
     drawnow (gl2ps_device{n}, strcat ('|',pipeline{n}));
   endfor
 
   if (! isempty (strfind (opts.devopt, "standalone")))
     opts.latex_standalone (opts);
   endif
 
